infrastructure/src/main/java/org/corfudb/infrastructure/FailureHandlerDispatcher.java

```diff
@@ -119,7 +119,17 @@ public void dispatchHandler(IFailureHandlerPolicy failureHandlerPolicy, Layout c
             corfuRuntime.invalidateLayout();
             if (corfuRuntime.getLayoutView().getLayout().equals(newLayout)) {
                 log.info("Failed node removed. New Layout committed = {}", newLayout);
+            } else {
+                log.warn("Layout recovered with a different layout = {}",
+                        corfuRuntime.getLayoutView().getLayout());
             }
+
+            //TODO: Since sequencer reset is moved after paxos. Make sure the runtime has the latest
+            //TODO: layout view and latest client router epoch. (Use quorum layout fetch.)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)